src/openrct2/world/Scenery.h

```diff
@@ -188,12 +189,12 @@ enum
     PATH_BIT_FLAG_IS_QUEUE_SCREEN = 1 << 8
 };
 
-enum
+enum class PathBitDrawType : uint8_t
 {
-    PATH_BIT_DRAW_TYPE_LIGHTS,
-    PATH_BIT_DRAW_TYPE_BINS,
-    PATH_BIT_DRAW_TYPE_BENCHES,
-    PATH_BIT_DRAW_TYPE_JUMPING_FOUNTAINS
+    Lights,
+    Bins,
+    Benches,
+    Jumping_Fountains
```

Comment:
Can you remove the underscore and all of these should really be singular as they represent a single object.

Reply:
Done